### PR TITLE
update instances of 'execution' and variations

### PR DESCRIPTION
**Pull request by Bradben**

### CODE REVIEW

**Review by KittyYeungQ — APPROVED** [review]

File: articles/user-guide/quickref/iqsharp.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ no-loc: ['Q#', '$$v']
 ### Azure Quantum integration
 
 - [`%azure.connect`](xref:microsoft.quantum.iqsharp.magic-ref.azure.connect): Connects to an Azure Quantum workspace or displays current connection status.
-- [`%azure.execute`](xref:microsoft.quantum.iqsharp.magic-ref.azure.execute): Executes a job in an Azure Quantum workspace.
+- [`%azure.execute`](xref:microsoft.quantum.iqsharp.magic-ref.azure.execute): Runs a job in an Azure Quantum workspace.
```
Comment (rmshaffer) [inline articles/user-guide/quickref/iqsharp.md:30]: "Submits a job to an Azure Quantum workspace and waits for completion." Similar update should also be made at https://github.com/microsoft/iqsharp/blob/740965405b813105b80e5b57d3714be9f0842f99/src/AzureClient/Magic/ExecuteMagic.cs#L32.

File: articles/user-guide/index.md [hunk 2]
```diff
@@ -48,13 +48,13 @@ In the different topics of this guide, we detail the core concepts of the Q# lan
 
 ### Quantum Simulators and Resource Estimators
 
-- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general execution model between host programs and target machines.
+- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general run model between host programs and target machines.
```
Comment (rmshaffer) [inline articles/user-guide/index.md:51]: I think ["execution model"](https://en.wikipedia.org/wiki/Execution_model) communicates a concept that "run model" doesn't.

File: articles/user-guide/machines/index.md [hunk 3]
```diff
@@ -17,15 +17,15 @@ Quantum simulators are software programs that run on classical computers and act
 
 ## The Quantum Development Kit (QDK) quantum simulators
 
-The quantum simulator is responsible for providing implementations of quantum primitives for an algorithm. This includes primitive operations such as `H`, `CNOT`, and `Measure`, as well as qubit management and tracking. The QDK includes different classes of quantum simulators representing different execution models for the same quantum algorithm. 
+The quantum simulator is responsible for providing implementations of quantum primitives for an algorithm. This includes primitive operations such as `H`, `CNOT`, and `Measure`, as well as qubit management and tracking. The QDK includes different classes of quantum simulators representing different run models for the same quantum algorithm.
```
Comment (rmshaffer) [inline articles/user-guide/machines/index.md:20]: I think ["execution model"](https://en.wikipedia.org/wiki/Execution_model) communicates a concept that "run model" doesn't.

File: articles/user-guide/machines/qc-trace-simulator/index.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ The QDK includes five tools that you can use with the quantum trace simulator to
 |-----| -----|
 |[Distinct inputs checker](xref:microsoft.quantum.machines.qc-trace-simulator.distinct-inputs) |Checks for potential conflicts with shared qubits |
 |[Invalidated qubits use checker](xref:microsoft.quantum.machines.qc-trace-simulator.invalidated-qubits)  |Checks if the program applies an operation to a qubit that is already released |
-|[Primitive operations counter](xref:microsoft.quantum.machines.qc-trace-simulator.primitive-counter)  | Counts the number of primitive executions used by every operation invoked in a quantum program  |
```
Comment (rmshaffer) [inline articles/user-guide/machines/qc-trace-simulator/index.md:84]: Why "primitive processes"? Can this just be "primitives"?

File: articles/overview/quantum-computers-and-simulators.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Quantum computers face a multitude of challenges to operate correctly. Error cor
 
 ## Quantum simulators
 
-Quantum simulators that run on classical computers allow you to simulate the execution of quantum algorithms on a quantum system.  Microsoft’s Quantum Development Kit (QDK) includes a full-state vector simulator along with other specialized quantum simulators.
+Quantum simulators that run on classical computers allow you to simulate the computation of quantum algorithms on a quantum system.  Microsoft’s Quantum Development Kit (QDK) includes a full-state vector simulator along with other specialized quantum simulators.
```
[the remark this answers is not in the document]
Comment (Bradben) [reply articles/overview/quantum-computers-and-simulators.md:29]: all good feedback, @rmshaffer, thanks - I'll fix in new PR today.

[the remark this answers is not in the document]
Comment (rmshaffer) [reply articles/overview/quantum-computers-and-simulators.md:29]: No worries, thanks @Bradben! Sorry for the extra work, just wanted to point out these few things.

**Pull request merged**